components/TopBar.tsx

```diff
@@ -10,6 +10,19 @@ import { deleteCookie } from "cookies-next";
 
 const TopBar = () => {
   const { userName } = useAuth();
+  const formatUserName = (userName: string) => {
+    const names = userName.split(' ');
+
+    const formattedFirstName = names[0].charAt(0).toUpperCase() + names[0].slice(1).toLowerCase();
+
+    const formattedLastName = names.length > 1 ? names[1].charAt(0).toUpperCase() + names[1].slice(1).toLowerCase() : '';
```

Comment:
what if names.length > 2? are we sure there will only be one space

Reply:
Yeah this works if user has only `first_name` and `last_name` and no `middle_name`. I've updated the code.